cfme/control/explorer/actions.py

```diff
@@ -181,6 +181,7 @@ def create(self):
             "remove_tag": self.remove_tag,
             "run_ansible_playbook": self.run_ansible_playbook
         })
+        self.browser.plugin.ensure_page_safe()
```

Comment:
Shouldn't the .click() ensure the page is safe? @mfalesni???

Reply:
Widgetastic's click calls ensure_page_safe at the end.

Reply:
So... does this mean we might want to add it at the beginning as well? Before clicking that is.

Reply:
@psav, added appropriate todo

Reply:
@jkrocil Element lookups do check as well, so we check before and after.

Reply:
So why is the `self.browser.plugin.ensure_page_safe()` required?

Reply:
I suspect it added some delay.